src/main/resources/spring/camel-context.xml

```diff
@@ -233,10 +232,14 @@
                     <to uri="direct:check-rbac-permissions" />
                     <!-- RBAC check end-->
 
-                    <to uri="direct:to-paginationBean"/>
-                    <to uri="direct:to-sortBean"/>
+                    <setHeader headerName="sort_by_defaults">
+                        <constant>provider,datacenter,cluster,vmName</constant>
+                    </setHeader>
+                    <to uri="direct:add-pageable-header"/>
```

Comment:
I would see more encapsulated and responsabilities better splited if it's the Service who is responsible of creating the Pageable object, as it was before.
wdyt ?

Reply:
@jonathanvila I've removed `<to uri="direct:add-pageable-header"/>` so the camel routes remain the same as it is right now in the master branch. The definition and creation of the object `Pageable` is made inside the services. What changed is just the name of the fields of `PageBean`; in the past, it used `page, size` and now it uses `offset, limit`

Reply:
+1